src/main/resources/application.conf

```diff
@@ -0,0 +1,12 @@
+dynamodb {
```

Comment:
Could you rename the variable names to remove the "piece". Apparently this is incorrect terminology. We'll update the DyanomDb table later.

So it would be:
```
...
referenceCounter = "pieceCounter"
referenceCounter = ${?REFERENCE_COUNTER}
...
```
The corresponding Terraform will also need to be updated.